mir_eval/separation.py

```diff
@@ -337,6 +497,51 @@ def _bss_decomp_mtifilt(reference_sources, estimated_source, j, flen):
     return (s_true, e_spat, e_interf, e_artif)
 
 
+def _bss_decomp_mtifilt_images(reference_sources, estimated_source, j, flen,
+                               Gj=None, G=None):
+    """Decomposition of an estimated source image into four components
+    representing respectively the true source image, spatial (or filtering)
+    distortion, interference and artifacts, derived from the true source
+    images using multichannel time-invariant filters.
+    Adapted version to work with multichannel sources.
+    Improved performance can be gained by passing Gj and G parameters.
```

Comment:
In that case, maybe give a quick note as to what they are/do here since those are not descriptive names?